server/configs/config.js

```diff
@@ -1 +1,17 @@
-export const PORT = process.env.PORT || 8000;
+module.exports = {
+  PORT: process.env.PORT || 8000,
+  google: {
+    callbackURL: 'http://localhost:8000/auth/google/redirect',
```

Comment:
I think there's a convention to use all caps name for constants. @rishabh1403 can you confirm?